package.json

```
@@ -11,8 +15,13 @@
   "gxVersion": "0.14.0",
   "language": "go",
   "license": "MIT",
-  "name": "dir-index-html",
   "releaseCmd": "git commit -a -m \"gx publish $VERSION\"",
-  "version": "1.0.4"
+  "scripts": {
+    "build": "npm run build:clean && npm run build:remove-style-links && npm run build:minify-wrap-css && npm run build:combine-html-css && npm run build:remove-unused",
+    "build:clean": "rm -rf ./dist/*",
+    "build:remove-style-links": "sed '/<link rel=\"stylesheet\"/d' ./src/dir-index.html > ./dist/base-html.html",
+    "build:minify-wrap-css": "(echo \"<style>\" && cat ./src/icons.css ./src/style.css | tr -d \"\t\n\r\" && echo \"</style>\") > ./dist/minified-wrapped-style.html",
```

> Yep, the idea here was to keep the whole thing as lightweight as possible

👍 

In that case my only nit is the following suggestion:

```suggestion
    "build:minify-wrap-css": "(echo \"<style>\" && cat ./src/icons.css ./src/style.css | tr -d \"\t\n\r\" && echo -e \"\\n</style>\") > ./dist/minified-wrapped-style.html",
```

---

oh! good catch 😊